symfony/console/3.3/bin/console

```diff
@@ -12,10 +12,13 @@ set_time_limit(0);
 require __DIR__.'/../vendor/autoload.php';
 
 if (!class_exists(Application::class)) {
-    throw new \RuntimeException('You need to add "symfony/framework-bundle" as a Composer dependency.');
+    throw new \RuntimeException('Application class does not exist. You need to add "symfony/framework-bundle" as a Composer dependency.');
```

Comment:
I would remove this addition as it does not add anything useful for the user. Nobody knows what this `Application` class is and why it is useful (and nobody should care to be honest). Moreover this error message should never be displayed anyway.

Reply:
Done.